test/fbe/test_github_graph.rb

```diff
@@ -91,4 +100,22 @@ def test_gets_total_commits_of_repo
     result = g.total_commits('zerocracy', 'baza', 'master')
     assert(result.positive?)
   end
+
+  def test_get_fake_empty_conversations
+    WebMock.disable_net_connect!
+    global = {}
```

Comment:
@Suban05 `global` and `options` look as [redundant vars](https://www.yegor256.com/2015/09/01/redundant-variables-are-evil.html)